src/main/java/com/fabriccommunity/spookytime/entity/SpookyCactusEntity.java

```diff
@@ -0,0 +1,136 @@
+package com.fabriccommunity.spookytime.entity;
+
+import java.util.List;
+
+import com.fabriccommunity.spookytime.block.SpookyCactusBlock;
+import com.fabriccommunity.spookytime.registry.SpookyBlocks;
+
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.EntityDimensions;
+import net.minecraft.entity.EntityPose;
+import net.minecraft.entity.EntityType;
+import net.minecraft.entity.ai.goal.WanderAroundFarGoal;
+import net.minecraft.entity.ai.goal.Goal.Control;
+import net.minecraft.entity.attribute.EntityAttributes;
+import net.minecraft.entity.damage.DamageSource;
+import net.minecraft.entity.data.DataTracker;
+import net.minecraft.entity.data.TrackedData;
+import net.minecraft.entity.data.TrackedDataHandlerRegistry;
+import net.minecraft.entity.mob.MobEntityWithAi;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.predicate.entity.EntityPredicates;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.ViewableWorld;
+import net.minecraft.world.World;
+
+@SuppressWarnings("unchecked")
+public class SpookyCactusEntity extends MobEntityWithAi {
+	public static final TrackedData<Integer> CACTUS_HEIGHT;
+	public BlockPos landPos;
+	public int age;
+	public int hop;
+
+	static {
+		CACTUS_HEIGHT = DataTracker.registerData(SpookyCactusEntity.class, TrackedDataHandlerRegistry.INTEGER);
```

Comment:
I would just same-line this unless there's a particular reason for the static block.